2주차/1620.py

```diff
@@ -0,0 +1,18 @@
+#1620_나는야 포켓몬 마스터 이다솜
+import sys
+
+n, m = map(int, input().split())
+poke_num={}
+poke_name={}
+
+for i in range(1, n+1):
+    a=sys.stdin.readline().strip()
+    poke_num[i]=a
+    poke_name[a]=i
+
+for _ in range(m):
+    p=sys.stdin.readline().strip()
+    if p.isdigit()==True:
```

Comment:
```suggestion
    if p.isdigit():
```
isdigit()의 반환값이 불리언이니까 굳이 ==True는 안써줘도 돼요